### DISCUSSION TITLE
Dark mode per site?

### DISCUSSION THREAD

**Discussion by stevedwray** [darkreader/darkreader#11419] [Q&A]
Hi,
I'm trying to figure out if this extension is intended to be able to turn dark mode on or off for specific sites.
Ie; 
I go to a site, its bright, I turn dark mode on and it works fine. I want to use dark mode for this site in future.
I go to a different site, its bright, I turn dark mode on and it doesn't work so well for this site. I don't want to use dark mode for this site in future.

When I'm using this extension, and turn dark mode on for one site, then go to another tab, I find that dark mode is already on for the other tab. I'd prefer to decide, site by site, as I use them, whether I want them to have dark mode enabled or not, and to have this decision be persistent.
Is that something this extension can do?

Thanks

**Comment by sc22t2b** [comment]
Yes it is something this extension can do, you can also configure it to work with a shortcut in chrome://extensions/shortcuts or you can click on the extension in the extensions menu and toggle it in the upper left of the pop up menu.

You can also view which sites you have toggled and there is actually a global list of sites that the community have confirmed to be already dark mode like monkeytype.com.
